dandiapi/api/models/version.py

```diff
@@ -165,7 +166,41 @@ def strip_metadata(cls, metadata):
             'publishedBy',
             'manifestLocation',
         ]
-        return {key: metadata[key] for key in metadata if key not in computed_fields}
+        stripped = {key: metadata[key] for key in metadata if key not in computed_fields}
+
+        # Ensure specific fields not modified
```

Comment:
I think this is a "what-not-why" comment (to borrow a saying from @danlamanna :slightly_smiling_face:). It's clear from reading the code that it is programatically removing certain fields, but I think the comment would be more useful if it explained _why_ we're excluding those fields from being modified, as imo it's not clear at first glance that it's at all related to embargo.